standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java

```diff
@@ -10740,7 +10740,7 @@ private static String getServerHostName() throws Exception {
     }
   }
 
-  private static void cleanupRawStore() {
+  static void cleanupRawStore() {
```

Comment:
This method is exclusively doing all the operations in HMSHandler. Does it make sense to have this method here?

Reply:
I'm not sure, I'll move it to `HMSHandler`. Thanks!